bpf/process/types/basic.h

```diff
@@ -879,37 +887,62 @@ filter_char_buf(struct selector_arg_filter *filter, char *args, int value_off)
 struct string_buf {
 	__u32 len;
 	char buf[];
-};
+} __attribute__((packed));
```

Comment:
why not to put it under one struct file_buf, there's no other usage for string_buf AFAICS

Reply:
I incorrectly figured that string_buf might end up getting used for the more general string cases in the future.